docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -36,82 +36,91 @@ loader: io.papermc.testplugin.TestPluginLoader
 
 ### Dependency Declaration
 
-Dependency declaration is specified a bit differently than Bukkit plugins, as you are
-able to define if a dependency is required during bootstrapping. This means that any bootstrapping
-logic for your dependencies will be run before yours is run. 
+Paper Plugins change how to declare dependencies in your `paper-plugin.yml`:
+
 ```yml
 dependencies:
-  - name: DependencyName
-    required: true
-    bootstrap: true
+  bootstrap:
+    # Lets say that RegistryPlugin has some registry elements that this plugin requires.
```

Comment:
What is a registry element?

Reply:
Hmm, I'm not really sure how to best describe why a plugin would need to do this *now* without relying on terms from future API. Is there even a use currently to define a bootstrap load order dependency?

Reply:
Correct, that's what I was going for when it comes to that. Although, it's hard since bootstrap still doesn't do anything so it's kind of just all over the place.